plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/AbstractDBMediatorDeserializer.java

```diff
@@ -22,6 +23,7 @@
 import org.wso2.developerstudio.eclipse.gmf.esb.SqlResultMapping;
 import org.wso2.developerstudio.eclipse.gmf.esb.SqlStatement;
 import static org.wso2.developerstudio.eclipse.gmf.esb.EsbPackage.Literals.*;
+import static org.wso2.developerstudio.eclipse.gmf.esb.Constants.*;
```

Comment:
Remove unused imports.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804